packages/shopping/src/models/product.model.ts

```diff
@@ -1,22 +1,17 @@
-// Copyright IBM Corp. 2018. All Rights Reserved.
-// Node module: loopback4-example-shopping
-// This file is licensed under the MIT License.
-// License text available at https://opensource.org/licenses/MIT
```

Comment:
Please preserve the license boilerplate header.

Reply:
👍